extensions/ql-vscode/src/vscode-tests/minimal-workspace/sourcemap.test.ts

```diff
@@ -0,0 +1,71 @@
+import { fail } from 'assert';
+import { commands, Selection, window, workspace } from 'vscode';
+import * as path from 'path';
+import * as assert from 'assert';
+import { expect } from 'chai';
+import { tmpDir } from '../../helpers';
+import * as fs from 'fs-extra';
+
+export function run() {
+  /**
+   * Integration tests for queries
+   */
+  describe('SourceMap', function() {
+    this.timeout(200000);
```

Comment:
Quite a large timeout. Is this necessary?

Reply:
Reduced. That was just to avoid a timeout when debugging.